scripts/index.js

```diff
@@ -0,0 +1,73 @@
+// Variables that are representing Elements
+let btnAdd = document.getElementById('add');
+let btnSubtract = document.getElementById('subtract');
+let tempInput = document.getElementById('temp_id');
+let landscape = document.getElementById('landscape');
+let skiesInput = document.getElementById('sky-select');
+let skiesLandscape = document.getElementById('sky-landscape');
+
+
+
+
+
+function countUp(element) {
+    element.value = parseInt(element.value) + 1;
+}
+
+function countDown(element) {
+    element.value = parseInt(element.value) - 1;
+}
+
+function determineColor() {
+    if (tempInput.value >= 80) {
+        tempInput.style.color = 'red';
+    } else if (tempInput.value >= 70 && tempInput.value <= 79) {
+        tempInput.style.color = 'orange';
+    } else if (tempInput.value >= 60 && tempInput.value <= 69) {
+        tempInput.style.color = 'yellow';
+    } else if (tempInput.value >= 50 && tempInput.value <= 59) {
+        tempInput.style.color = 'green';
+    } else if (tempInput.value <= 49) {
+        tempInput.style.color = 'teal';
+    }
```

Comment:
In general don't set styles directly, instead set the element's class name.  Otherwise you're mixing roles between the behavior (JS) and presentation (CSS).